prodigy.el

```diff
@@ -1274,8 +1274,10 @@ SIGNINT signal."
 ;;;; View mode functions
 
 (defun prodigy-strip-ctrl-m (output)
-  "Strip 
 line endings from OUTPUT."
-  (s-replace "
" "" output))
+  "Strip 
+ line endings from OUTPUT."
+  (s-replace "
+" "" output))
```

Comment:
Maybe. Can you try to force push the commit or something like that to remove it?